test/java/org/opendatakit/briefcase/push/RemoteServerTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2018 Nafundi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.opendatakit.briefcase.push;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Test;
+import org.opendatakit.briefcase.reused.http.FakeHttp;
+import org.opendatakit.briefcase.reused.http.Query;
+
+public class RemoteServerTest {
```

Comment:
I don't think this test is valid and I think I led you astray with my curl command. You'll need to pass in a "X-OpenRosa-Version: 1.0" header..

`curl --header "X-OpenRosa-Version: 1.0" https://opendatakit.appspot.com/formList`

Your output will look like this:
```xml
<xforms xmlns="http://openrosa.org/xforms/xformsList">
<xform><formID>CascadingSelect</formID>
<name>Cascading Select Form</name>
<majorMinorVersion>2012072302</majorMinorVersion>
<version>2012072302</version>
<hash>md5:29a492009d35c1515886ddd307521819</hash>
<downloadUrl>https://opendatakit.appspot.com/formXml?formId=CascadingSelect</downloadUrl>
</xform>
<xform><formID>widgets</formID>
<name>Widgets</name>
<majorMinorVersion></majorMinorVersion>
<version></version>
<hash>md5:0eacc6333449e66826326eb5fcc75749</hash>
...
```

Reply:
OK, I've added the header and changed the test.